### ISSUE TITLE
Re-enable support for min_tls_version and allow_public_blob_access in US Government

### ISSUE THREAD

**Issue by adamday2** [hashicorp/terraform-provider-azurerm#9128]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

The referenced pull request disabled support for min_tls_version and allow_public_blob_access in US Gov environments due to lack of support. These features appear to now be available in the Azure portal in US Gov and the Azure CLI, and this change should now be reverted.

Trimmed result of ` az storage account show --ids <storage account>` after configuring via portal:
```
  "allowBlobPublicAccess": false,
  "location": "usgovvirginia",
  "minimumTlsVersion": "TLS1_2",
```

### New or Affected Resource(s)

* azurerm_storage_account

### References

* https://github.com/terraform-providers/terraform-provider-azurerm/pull/8092

**Comment by ruandersMSFT** [comment]
Previously filtered in issue #8092

**Comment by ruandersMSFT** [comment]
PR #9539 has been compiled and tested, have successfully created and modified a storage account in US Gov Cloud.

![image](https://user-images.githubusercontent.com/16927960/100533469-fd1e4280-31c1-11eb-896b-7a8f5136aa9b.png)

**Comment by ruandersMSFT** [comment]
@tombuildsstuff  Please close issue, fix pulled in #9540, fix released in AzureRM v2.40.0.

**Comment by jsgarciad** [comment]
This issued already was re-enable through this PR -> https://github.com/terraform-providers/terraform-provider-azurerm/pull/9540 

Use the azurerm provider version >2.40.0 and enjoy this great feature for government :D !
